### PR TITLE
1455 vacancies

### PR DESCRIPTION
**Pull request by tjamesmac**
### What is the context of this PR?
Added the new Vacancies surveys to test.
There should be three variants of _0006:
- 182
- 183
- 184

And a single variant of _0005:
- 185

### How to review 
Launch the surveys with survey-runner and make sure to supply an employmentDate in date format.

### Checklist

* [ ] q_code field for vacancies question == 10
* [ ] q_code field for comment question == 146
* [ ] Values for q_codes are string
* [ ] max_length value for comment textArea !== null
* [ ] title fields for 182/183/184 is "Quarterly Vacancy Survey"
* [ ] title fields for 185 is "Monthly Vacancy Survey"

### CODE REVIEW

**Review by martyncolmer — APPROVED** [review]
Good job

File: data/en/182_0006.json [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                                    "type": "General",
+                                    "answers": [
+                                        {
+                                            "id": "answerb0ca3f20-34fc-48ff-982e-2bb047afe7df",
+                                            "mandatory": true,
+                                            "type": "Number",
+                                            "label": "Number of vacancies",
+                                            "description": "If an exact figure is not available, provide an informed estimate. If you did not have any vacancies, enter 0.",
+                                            "decimal_places": 0,
+                                            "q_code": 10
```
Comment (MattElcock) [inline data/en/182_0006.json]: `10` should be a String, not a number

**Review by MattElcock — CHANGES_REQUESTED** [review]
Looks good, nice one. Just one small change.

**Comment by MattElcock** [comment]
Running `yarn format` will fix your Travis build error

**Review by MattElcock — APPROVED** [review]
Looks good!

**Pull request closed without merging**